packages/esm-patient-clinical-view-app/jest.config.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This file does not need to be repeated per package. Its just declared once at the root.